<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRentalWindow</class>
 <widget class="QDialog" name="EditRentalWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Rental</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/res/data/icons/edit.webp</normaloff>:/res/data/icons/edit.webp</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#frameRent
{
        background-color:#333;
}

QLabel#RentCarlbl
{
        font-size:20px;
        background-color:#222;
        border-radius:12px;
}

QLabel#NumOfDays, QLabel#PickStartDay
{
        font-size:11px;
        background-color:#222;
        border-radius:12px;
}

QCalendarWidget#calendarWidget
{
         color: rgb(180, 180, 180);
      background-color: black;
      selection-background-color: rgb(233, 60, 44 );
      selection-color: rgb(255, 255, 255 );
}
QSpinBox#spinBox
{
        background-color:#222;
font-size:11px;
}

QCalendarWidget#calendarWidget
{
         color: rgb(180, 180, 180);
      background-color: black;
      selection-background-color: rgb(233, 60, 44 );
      selection-color: rgb(255, 255, 255 );
}
QPushButton#RentRequ
{
        background-color:#222;
        font-size:11px;
}

QPushButton#RentRequ:hover
{
        background-color:#444;
        font-size:11px
}

QLabel#CalWarning
{
        font-size:11px;
                color:red;
}</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>280</width>
     <height>156</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	  color: rgb(180, 180, 180);
      background-color: #111;
      selection-background-color: rgb(233, 60, 44 );
      selection-color: rgb(255, 255, 255 );
          font-size:10px;
alternate-background-color: rgb(128, 128, 128);</string>
   </property>
  </widget>
  <widget class="QLabel" name="PickStartDay">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> Pick a starting day :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RentRequ">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>520</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Rental</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameRent">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>591</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="RentCarlbl">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>   Edit Rental :</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="NumOfDays">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Pick the number of Days you wish to rent the car:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="CalWarning">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <zorder>frameRent</zorder>
  <zorder>calendarWidget</zorder>
  <zorder>PickStartDay</zorder>
  <zorder>RentRequ</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
